distro/wildfly/compatibility-subsystem27/pom.xml

```diff
@@ -0,0 +1,196 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.camunda.bpm.wildfly</groupId>
+    <artifactId>camunda-wildfly</artifactId>
+    <version>7.20.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>camunda-bpm-wildfly-27-subsystem-compatibility</artifactId>
+
+  <name>Camunda Platform - Wildfly 27 subsystem compatibility</name>
+
+  <properties>
+    <version.wildfly>27.0.1.Final</version.wildfly>
+    <version.wildfly.core>19.0.1.Final</version.wildfly.core>
+  </properties>
+
+  <dependencies>
+      <dependency>
+        <groupId>org.camunda.bpm.wildfly</groupId>
+        <artifactId>camunda-wildfly-subsystem</artifactId>
+        <version>7.20.0-SNAPSHOT</version>
```

Comment:
🔧 We usually use `${project.version}` instead. Any specific reason for using the version explicitly here?